apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -88,6 +89,14 @@ class LoginTypePicker extends Component {
     const withClever =
       providers && providers.includes(OAuthSectionTypes.clever);
     const hasThirdParty = withGoogle | withMicrosoft | withClever;
+    const currentUser = getStore().getState().currentUser;
+    const inUSA =
+      ['US', 'RD'].includes(currentUser.countryCode) ||
+      !!currentUser.usStateCode;
+    const showStudentsToSectionPermissionWarning =
+      inUSA &&
+      currentUser.isTeacher &&
+      experiments.isEnabledAllowingQueryString(experiments.CPA_EXPERIENCE);
```

Comment:
Should this be an OR statement? My understanding is that the experiment is a way for us to override the conditions for the banner to appear.

Reply:
cc @wilkie

Reply:
Yep. Should be an `||` as Mario suggests, I think.

With the `&&` it would only show the banner if the `cpa_experience` DCDO flag is set (or you set it in the URL or via `enableExperiments=`). That flag is not set in production, so it wouldn't show at all. I think the intent is to actually roll this out.

That flag should just force you into a situation where things are acting like they would if you were a targeted user and the entire feature is live.

I think I like updating it to an `||` so it's live on deploy that it can still be forced on for testing via that flag.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/58497/commits/26a8e90f87a4f12377cdfe2d65b9ec8442c77107